src/Modal/ModalFooter/index.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+import * as React from "react";
+import styled from "styled-components";
+
+import media from "../../utils/media";
+import defaultTokens from "../../defaultTokens";
+import { StyledButton } from "../../Button";
+
+import type { Props } from "./index";
+
+const StyledChild = styled.div`
+  flex: ${({ flex }) => flex};
+  box-sizing: border-box;
+  padding-right: ${({ theme }) => theme.orbit.spaceMedium};
+
+  ${media.desktop`
+    flex: none;
+  `};
+`;
+
+StyledChild.defaultProps = {
+  theme: defaultTokens,
+};
+
+const StyledModalFooter = styled.div`
+  display: flex;
+  position: ${({ fixed }) => fixed && "fixed"};
+  bottom: 0;
+  width: 100%;
+  background-color: ${({ theme }) => theme.orbit.paletteWhite};
+  // TODO: create token paddingModalFooter
+  padding: ${({ theme, fixed }) =>
+    fixed ? theme.orbit.spaceMedium : `0 ${theme.orbit.spaceMedium} ${theme.orbit.spaceMedium}`};
+  box-sizing: border-box;
+  box-shadow: ${({ fixed }) => fixed && `0 -2px 4px 0 rgba(23, 27, 30, 0.1)`};
+  // TODO: create token boxShadowActionableInverted
+
+  ${media.desktop`
+    position: static;
+    box-shadow: none;
+    justify-content: space-between;
+    // TODO: create token paddingModalFooterDesktop
+    padding: ${({ theme }) => `0 ${theme.orbit.spaceXXLarge} ${theme.orbit.spaceXXLarge}`};
+    border-bottom-left-radius: ${({ theme }) => theme.orbit.borderRadiusNormal};
+    border-bottom-right-radius: ${({ theme }) => theme.orbit.borderRadiusNormal};
+  `};
+
+  ${StyledChild}:last-of-type {
+    padding-right: 0;
+  }
+
+  ${media.mobileOnly`
```

Comment:
Please use mobileFirst and override

Reply:
This is the very edge case where I need to set `font-size` and `heigh` for mobile only. If I use the mobile first approach, I need to somehow redeclare these properties in `desktop`, but buttons can have `medium` and `large` sizes and you don't know what size was used.